src/pages/__tests__/Profile.page.test.js

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
I thought we were asked not to use `enzyme`, since it doesn't work well with React hooks?